github/issues_timeline_test.go

```diff
@@ -11,10 +11,20 @@ import (
 	"net/http"
 	"strings"
 	"testing"
+	"time"
 
 	"github.com/google/go-cmp/cmp"
 )
 
+func parseTime(t *testing.T, value string) time.Time {
```

Comment:
Honestly, there is really no need for this helper function, as there is no need to test the functionality of `time.Parse` in this repo since it is well-tested with each release of the Go compiler.

The vast majority of our tests use `&Timestamp{referenceTime}` although I do see some unfortunately deviations from this.

Let's please remove this helper function and use the existing `referenceTime` instead.

Reply:
Sounds good. I've pushed up those changes to use `&Timestamp{referenceTime}`!